### ISSUE TITLE
Can't install... says the package name is not the expected...

### ISSUE THREAD

**Issue by mpierangeli26** [files-community/Files#15736]
### Description

Can't install... says the package name is not the expected...to verify log pf AppX... The Files Classic installer file was downloaded from official site. Version 3.5.0.0
Something is wrong...

### Steps To Reproduce

1. Download Classic Installer from https://files.community/download
2. Execute the file
3. The error appears

### Requirements

- e.g. Fix crash that occurs when...
I try t install classic version

### Files Version

3.5.0.0

### Windows Version

Windows 11 22H2 Build 22621.3810

### Log File

Can't access

**Comment by Josh65-2201** [comment]
Thanks for the report, Please send a screenshot of the full error.

**Comment by msmacedo68** [comment]
Same here. See the screenshot:
![image](https://github.com/files-community/Files/assets/57114888/cf0574c6-df8d-484e-900e-a7c5ea9aa80b)
![2024-07-02_184138](https://github.com/files-community/Files/assets/57114888/f6857d7e-3776-4345-9430-53771e0e6dd3)

**Comment by Josh65-2201** [comment]
Please follow the instructions at https://files.community/docs/install#:~:text=starting%20an%20update.-,Steps%20required,-for%20the%20%E2%80%9Cclassic

**Comment by msmacedo68** [comment]
Hi Folks,
I've followed the instruction and completed the installations.
It wasn't without a little pain, tough.

- 1st, I've updated the Windows AppInstaller, booted the PC, but the installation still failed.
- 2nd, I've tried the PowerShell method. Failed again.
- Finally, I've uninstalled the previous version (3.2), booted the PC, and then the installation completed.

Thus, as far as I could see, the latest version will not update the previous installation, o matter what. It required the previous version "manually" removed.

BR!
